src/db.c

```diff
@@ -121,7 +124,8 @@ void updateKeysizesHist(redisDb *db, int didx, uint32_t type, uint64_t oldLen, u
  * expired on replicas even if the master is lagging expiring our key via DELs
  * in the replication link. */
 robj *lookupKey(redisDb *db, robj *key, int flags, dictEntry **deref) {
-    dictEntry *de = dbFind(db, key->ptr);
+    const int keySlot = getKeySlot(key->ptr);
+    dictEntry *de = dbFindWithKeySlot(db, key->ptr, keySlot);
```

Comment:
```suggestion
    dictEntry *de = dbFindWithKeySlot(db, key->ptr, key_slot);
```
perfer to use underscore naming.
there are a few others like it.